File: revel-app/app/models/entities/Employee.go
package entities

//Employee структура сущности сотрудника
type Employee struct {
	ID         int64  `json:"ID"`
	Firstname  string `json:"first name"`
	Lastname   string `json:"last name"`
	Patronymic string `json:"patronymic"`
	Position   string `json:"position"`
	Email      string `json:"email"`
	Phone      string `json:"phone"`
	IDUser     int64  `json:"ID user"`
}

File: revel-app/public/js/src/models/entities/Employee.js
export class Employee{
    /**
     * 
     * @param {number} id 
     * @param {string} firstname 
     * @param {string} lastname 
     * @param {string} patronymic 
     * @param {string} position 
     * @param {string} email 
     * @param {string} phone 
     * @param {string} id_user 
     */
    constructor(id,firstname, lastname, patronymic, position, email, phone, id_user){
        this.ID = id
        this.firstname = firstname
        this.lastname = lastname
        this.patronymic = patronymic
        this.position = position
        this.email = email
        this.phone = phone
        this.id_user = id_user
    }
}